### PR TITLE
Add framework and simple test for navigation band

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,17 +53,19 @@
   },
   "main": "index.js",
   "dependencies": {
+    "@brightspace-ui/visual-diff": "^0.1.0",
```
Comment (jeroach) [inline package.json]: Going to move these to `dev-dependencies`

Comment (vix633) [reply package.json]: Need puppeteer as well

File: package.json [hunk 2]
```diff
@@ -37,6 +38,7 @@
     "lie": "^3.3.0",
     "merge-stream": "^1.0.1",
     "polymer-cli": "^1.9.6",
+    "puppeteer": "^1.20.0",
```
Comment (vix633) [inline package.json]: ye

**Comment by dbatiste** [comment]
I think it would be preferable to hold off on using `visual-diff` in more places until we have @margaree's latest visual-diff changes complete since it will be a breaking change.  These changes will store the goldens. I think this will be fairly soon.  There will likely be some follow-up changes to the "bot" (GitHub app).

**Comment by DiljotSG** [comment]
[This PR](https://github.com/BrightspaceUI/navigation/pull/84) should be merged next (and after @margaree's changes are done).

**Comment by DiljotSG** [comment]
@jeroach I merged the required changes to the travis config to this branch. Now we should see the vdiff tests fail and then the option to regenerate goldens will appear and will allow this build to pass.

File: package.json [hunk 3]
```diff
@@ -12,11 +12,15 @@
     "lint:wc": "polymer lint -i \"./*\" \"test\" \"demo\"",
     "test:polymer:local": "polymer test --skip-plugin sauce",
     "test:polymer:sauce": "polymer test --skip-plugin local",
-    "test": "npm run lint && npm run test:polymer:local"
+    "test": "npm run lint && npm run test:polymer:local",
+    "test:diff": "mocha ./test/**/*.visual-diff.js -t 10000",
+    "test:diff:golden": "mocha ./test/**/*.visual-diff.js -t 10000 --golden",
```
Comment (DiljotSG) [inline package.json]: We had timeout issues on `activities` so we bumped this timeout up to `40000`. Might be worth doing here.

**Comment by jeroach** [comment]
> @jeroach I merged the required changes to the travis config to this branch. Now we should see the vdiff tests fail and then the option to regenerate goldens will appear and will allow this build to pass.

Thanks :)

**Comment by DiljotSG** [comment]
@jeroach I will approve this once you regenerate the goldens (otherwise my review will be dismissed when the bot pushes the goldens) 👍

**Comment by jeroach** [comment]
> @jeroach I will approve this once you regenerate the goldens (otherwise my review will be dismissed when the bot pushes the goldens) 👍

Sounds good, thanks

**Pull request merged**